packages/back-end/src/services/experiments.ts

```diff
@@ -2207,7 +2207,7 @@ export async function getLinkedFeatureInfo(
 
     const draftMatches =
       revisions
-        .filter((r) => r.status === "draft")
+        .filter((r) => r.status === "draft" || r.status === "pending-review")
```

Comment:
I think there are more draft-like statuses for approved and changes-requested.

Reply:
thanks, added